### PR TITLE
chore: update tools, use new generators

### PR DESCRIPTION
**Pull request by AlekSi**
# Pull Request

## What? (description)

chore: update tools, use new generators

## Why? (reasoning)

To stay current.

See also https://github.com/talos-systems/tools/pull/128.

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [x] you generated documentation (`make docs`)
- [x] you ran unit-tests (`make unit-tests`)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ REGISTRY_AND_USERNAME := $(IMAGE_REGISTRY)/$(USERNAME)
 DOCKER_LOGIN_ENABLED ?= true
 
 ARTIFACTS := _out
-TOOLS ?= ghcr.io/talos-systems/tools:v0.3.0-21-g0026740
+TOOLS ?= ghcr.io/aleksi/tools:v0.3.0-23-g00e42a3
```
Comment (AlekSi) [inline Makefile:12]: Should be updated when https://github.com/talos-systems/tools/pull/128 is merged

**Review by andrewrynhard — DISMISSED** [review]
Might be able to get rid of the maligned errors if the embedded structs go first. That is the convention we use everywhere else at least.

**Review by smira — APPROVED** [review]

**Comment by AlekSi** [comment]
While working on another PR, I received "service storage.StorageService is not defined" error which sounds like might be caused by that PR. Will investigate.

**Review by smira — APPROVED** [review]

**Pull request merged**